README.md

```diff
@@ -61,6 +61,12 @@ The following projects are official OpenJS Foundation projects. If you are inter
 * [Fastify](https://fastify.io)
 * [nvm](http://nvm.sh/)
 
+
+### OpenJS Collaboration spaces
+
+The following are official OpenJS Collaboration spaces. If you are interested in fostering collaboration with the 
+support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression]
+
```

Comment:
Should there be a note here that there are none? It seems weird for it to mention `The following are official OpenJS Collaboration spaces` but then have nothing blow

Reply:
@MylesBorins good catch, added.

Reply:
added